src/routes/Outbox.jsx

```diff
@@ -0,0 +1,78 @@
+// React Imports
+import React, { useContext, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Library Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Styling Imports
+import styled from 'styled-components';
+// Utility Imports
+import { getMessageTTL } from '../utils';
+// Context Imports
+import { OutboxMessageContext } from '../contexts';
+// Component Imports
+import PaginatedMessages from '../components/Messages/Pagination';
+
+/**
+ * Outbox Component - Component that generates Outbox section for users
+ * logged into a Solid Pod via Solid Session
+ *
+ * @memberof Outbox
+ * @name Outbox
+ */
+
+const Outbox = ({ loadMessages }) => {
+  const location = useLocation();
+
+  localStorage.setItem('restorePath', location.pathname);
+
+  const { session } = useSession();
+  const { outboxList, setOutboxList } = useContext(OutboxMessageContext);
+
+  const podUrl = session.info.webId.split('profile')[0];
+
+  // Handler function for refreshing PASS outbox
+  const handleOutboxRefresh = async () => {
+    const messagesInSolid = await getMessageTTL(session, 'Outbox', outboxList, podUrl);
+    messagesInSolid.sort((a, b) => b.uploadDate - a.uploadDate);
+    setOutboxList(messagesInSolid);
+  };
+
+  // Re-sorts messages upon outboxList updating
```

Comment:
For inbox, it's kind not really possible to automatically be updated outside of initial login or refresh, unless we use some kind of setInterval to refresh the list at a set time.

It's because messages in inbox is not dependent on the recipient. It's depending on the sender.

Reply:
Yeah updating inbox dynamically would require hooking up to Solid's notification system or something similar. That's a fairly involved networking process, and IMO can be left to another PR.